src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const ordersController = require('../controllers/orders.controller');
+
+/**
+ * @swagger
+ * /orders:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get all orders of specific user
```

Comment:
This is not crrect documantaion. `api/orders` will return ALL orders not for specific user.
PLeas read carefully documentation you provide